src/renderer/util/fileSystem.js

```diff
@@ -45,11 +45,13 @@ export const moveToRelativeFolder = async (cwd, imagePath, relativeName) => {
   //  - markdown file directory + relative directory name or
   //  - root directory + relative directory name
   const absPath = path.resolve(cwd, relativeName)
-  ensureDirSync(absPath)
-
   const dstPath = path.resolve(absPath, path.basename(imagePath))
   await fs.move(imagePath, dstPath, { overwrite: true })
-  return dstPath
+  ensureDirSync(absPath)
```

Comment:
Thanks for your contribution! Please redo the `ensureDirSync` call that the directory is created before the file is copied, as before. Otherwise LGTM.

Reply:
Thank you. I'll fix it.